test/tests/reporters/mesh_info/gold/mesh_info_out.json

```diff
@@ -4,6 +4,18 @@
     "time_steps": [
         {
             "reporters": [
+                {
+                    "object_name": "mesh_info",
+                    "type": "std::map<short, MeshInfo::SidesetInfo, std::less<short>, std::allocator<std::pair<short const, MeshInfo::SidesetInfo> > >",
+                    "value": null,
+                    "value_name": "local_sideset_elems"
+                },
+                {
+                    "object_name": "mesh_info",
+                    "type": "std::map<short, MeshInfo::SidesetInfo, std::less<short>, std::allocator<std::pair<short const, MeshInfo::SidesetInfo> > >",
+                    "value": null,
+                    "value_name": "local_sidesets"
+                },
```

Comment:
@aeslaughter is there a way to clean this up? On initial - this information isn't available.

Reply:
The JSON output just needs to check if the value is null. That could be added to #16691, which changes how the JSON output is handled.

Reply:
However, it might be nice to see that it is null. That will avoid the question, "why don't I see my output?" It will also help with scripts and future Peacock support. I would rather have the data null then non-existent, thus a spelling error is different than a null value error.

Reply:
Probably just need an option: "output_null ="